vulntotal/datasources/deps.py

```diff
@@ -0,0 +1,122 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and https://github.com/nexB/vulnerablecode/
+# The VulnTotal software is licensed under the Apache License version 2.0.
```

Comment:
Same nit as elsewhere, please use the new simpler header

Reply:
Fixed in https://github.com/nexB/vulnerablecode/pull/789/commits/84100766a5f6be7a4183ee7c6f165cc1924b3123